Uplift/Views/GymDetail/Facilities/PriceInformationCell.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  PriceInformationCell.swift
+//  Uplift
+//
+//  Created by Yana Sang on 10/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class PriceInformationCell: UICollectionViewCell {
+
+    struct Constants {
+        static let lineHeight: CGFloat = 20
```

Comment:
seems like this is only used in the `getHeight` function? if so lets just make it a `let` variable inside the function instead of putting it in this struct